app/imports/components/apply/applyLayout.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { PropTypes } from 'react';
+
+// Import apply parts
+import TextTest from '/imports/components/apply/text-test.jsx';
+
+import FormTest from '/imports/components/apply/form-test.jsx';
+
+const getApplySection = ( isLogged, user, apply ) => {
+  // Check if logged in
+  if( !isLogged  ) {
+    return <Accounts.ui.LoginForm />;
+  }
+
+  if( !!apply.section ) {
+    switch(apply.section) {
+      case 'text-test': return <TextTest />;
+      case 'form-test': return <FormTest />;
+      default: return ':('; // 404
+    };
+  }
+
+};
+
+export default ApplyLayout = ({ isLogged, user, apply }) => (
+  <section className="applyLayout">
+    { getApplySection(isLogged, user, apply) }
+  </section>
+);
+
+ApplyLayout.protoTypes = {
```

Comment:
ApplyLayout is the defined component in this file. Even tho this is a stateless component propTypes and contextTypes are supported.